core/templates/dev/head/domain/exploration/AnswerGroupObjectFactory.js

```diff
@@ -13,22 +13,24 @@
 // limitations under the License.
 
 oppia.factory('AnswerGroupObjectFactory', [function() {
-  var AnswerGroup = function(ruleSpecs, outcome) {
+  var AnswerGroup = function(ruleSpecs, outcome, correct) {
     this.ruleSpecs = ruleSpecs;
     this.outcome = outcome;
+    this.correct = correct;
   };
 
   AnswerGroup.prototype.toBackendDict = function() {
     return {
       rule_specs: this.ruleSpecs,
-      outcome: this.outcome
+      outcome: this.outcome,
+      correct: this.correct
     };
   };
 
   // Static class methods. Note that "this" is not available in
   // static contexts.
-  AnswerGroup.create = function(ruleSpecs, outcome) {
-    return new AnswerGroup(ruleSpecs, outcome);
+  AnswerGroup.create = function(ruleSpecs, outcome, correct) {
```

Comment:
You'll also have to change the places which call `AnswerGroup.create`, for example the `generateAnswerGroupsFromBackend` calls `AnswerGroupObectFactory.create`, so you'll need to pass in the `correct` parameter then.